Fix the uuid_type error in BLEUUIDBase and add some new functions to BLEAdapter

General-zjs opened the pull request:
When creating the BLEUUIDBase class, if tht vs_uuid_base  is none and the uuid_type is none, the value of uuid_type should be BLE_UUID_ TYPE_ VENDOR_ BEGIN





Signed-off-by: h1562075881 <1562075881@qq.com>

General-zjs committed:
Fix the uuid_type error in BLEUUIDBase

Signed-off-by: h1562075881 <1562075881@qq.com>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/NordicSemiconductor/pc-ble-driver-py?pullRequest=229) <br/>All committers have signed the CLA.

General-zjs committed:
add function 'scan', 'connect', 'ble_stack_init', 'ble_cfg_set'

Signed-off-by: h1562075881 <1562075881@qq.com>

General-zjs committed:
1.ble_adapter.EvtSync added function [call] to resolve issue #43, refer from issue #55.
2.ble_adapter optimizes the write_req and write_cmd functions to resolve an error when the length of a command sent is longer than att_mtu.
3.ble_driver._subversion_number_to_softdevice_info Adds new softdevice information
4. Add new print information to observers.
